src/views/EditorHome.vue

```diff
@@ -690,6 +702,7 @@ export default defineComponent({
       isCharacterOrderDialogOpenComputed,
       isDefaultStyleSelectDialogOpenComputed,
       isDictionaryManageDialogOpenComputed,
+      isEngineManageDialogOpenComputed,
```

Comment:
別に直さなくても全く問題ないのですが、気づいたので。。（なんとなく登場順にする感じです）
```suggestion
      isEngineManageDialogOpenComputed,
      isDictionaryManageDialogOpenComputed,
```